tests/beacon/types/test_casper_slashings.py

```diff
@@ -0,0 +1,24 @@
+from eth.beacon.types.casper_slashings import (
+    CasperSlashing,
+    SlashableVoteData,
+)
+
+
+def test_defaults_for_casper_votes(sample_slashable_vote_data_params):
```

Comment:
```suggestion
def test_defaults_for_slashable_vote_data(sample_slashable_vote_data_params):
```

Reply:
I guess you would like to separate `test_casper_slashings.py` into a `test_casper_slashing.py` and a `test_slashable_vote_data.py`?